src/store/reducers.js

```diff
@@ -43,6 +44,11 @@ function entityLoaded(state, { payload: { entitlements } } = { payload: {} }) {
                 title: 'Compliance',
                 name: 'compliance',
                 component: ComplianceTab
+            },
+            insights.chrome.isProd && isEntitled(entitlements && entitlements.smart_management) && {
```

Comment:
Why is there a check isProd? It would make sense to me to display it everywhere EXCEPT prod until we release to prod-stable.

Reply:
So we can seperate our proccesses and you can release at your own pace and so do we. I'm planning to promote this change to all envs and once you give us ACK we can remove this to enable it for all users.